erpnext/regional/doctype/lower_deduction_certificate/lower_deduction_certificate.py

```diff
@@ -24,3 +28,10 @@ def validate(self):
 			<= fiscal_year.year_end_date):
 			frappe.throw(_("Valid Upto date not in Fiscal Year {0}").format(frappe.bold(self.fiscal_year)))
 
+	def validate_supplier_against_section_code(self):
```

Comment:
Also check for dates in this. There can be two LDC certificate for a supplier and section code for non overlapping dates